dcrtimed/dcrtimed.go

```diff
@@ -585,12 +692,11 @@ func _main() error {
 			// CORS options
 			origins := handlers.AllowedOrigins([]string{"*"})
 			methods := handlers.AllowedMethods([]string{"GET", "OPTIONS", "POST"})
-			headers := handlers.AllowedHeaders([]string{"Content-Type"})
 
 			log.Infof("Listen: %v", listen)
 			listenC <- http.ListenAndServeTLS(listen,
 				loadedCfg.HTTPSCert, loadedCfg.HTTPSKey,
-				handlers.CORS(origins, methods, headers)(d.router))
+				handlers.CORS(origins, methods)(d.router))
```

Comment:
please explain this change.

Reply:
This was a mistake. Reverting this change.